### ISSUE TITLE
False positive on multiple computed property access

### ISSUE THREAD

**Issue by futpib** [dustinspecker/eslint-plugin-no-use-extend-native#124]
Code like this is reported:
```js
({
	foo: {
		bar: 1,
		buz: 2,
	},
}[i][j])
```

Quick way to reproduce:

```bash
echo '{}[i][j]' | yarn eslint --plugin no-use-extend-native --rule '{"no-use-extend-native/no-use-extend-native":"error"}' --stdin
```

```
1:3  error  Avoid using extended native objects  no-use-extend-native/no-use-extend-native
```

If I remove one level of property access (change to `{}[i]`), nothing is reported.

**Comment by dustinspecker** [comment]
Thank you!

Think I got a fix in place to prevent this false positive. I'll publish 0.4.1 after tests pass on CI.

**Comment by dustinspecker** [comment]
Published 0.4.1.
